pkg/api/task_info.go

```diff
@@ -25,7 +25,8 @@ func (t *TaskInfoCollectionResponse) SetMetadata(meta ResponseMetadata, links Li
 }
 
 type TaskInfoFilterData struct {
-	Status         string `query:"status" json:"status"`
-	Typename       string `query:"type" json:"type"`
-	RepoConfigUUID string `query:"repository_uuid" json:"repository_uuid"`
+	Status           string `query:"status" json:"status"`
+	Typename         string `query:"type" json:"type"`
+	RepoConfigUUID   string `query:"repository_uuid" json:"repository_uuid"`
+	ExcludeRedHatOrg bool   `json:"-" swaggerignore:"true"`
```

Comment:
I left this out of the API spec because it wasn't on the card, but it might useful there. Thoughts?

Reply:
+1 to adding it, i think it makes sense